graphql/tests/query/adminSettings/results.json

```diff
@@ -0,0 +1,35 @@
+{
+  "tests": [
+    {
+      "query_file": "no_permissions.graphql",
+      "test_user_id": "regular_user",
+      "result": {
+        "data": {
+          "adminSettings": null
+        },
+        "errors": [
+          {
+            "message": "User 'regular_user' lacks required admin permissions",
+            "path": [
+              "adminSettings"
+            ],
+            "extensions": {
+              "code": "INTERNAL_SERVER_ERROR"
```

Comment:
Oh sorry, I missed this in your other PR. We actually want a `FORBIDDEN` code here. Could you edit the directive in `resolver.go` so that it returns a `FORBIDDEN` code?